src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1062,7 +1069,47 @@ export default abstract class ComparisonStore
             return Promise.resolve([]);
         },
     });
+    @computed get alterationEnrichmentRowData(): AlterationEnrichmentRow[] {
+        if (
+            this.alterationsEnrichmentData.isComplete &&
+            this.alterationsEnrichmentAnalysisGroups.isComplete
+        )
+            return getAlterationRowData(
+                this.alterationsEnrichmentData.result!,
+                this.resultsViewStore
+                    ? this.resultsViewStore.hugoGeneSymbols
+                    : [],
+                this.alterationsEnrichmentAnalysisGroups.result!
+            );
+        return [];
+    }
 
+    @computed get genesOfInterestForPathwayMapper(): AlterationEnrichmentRow[] {
+        const newGenes: string[] = this.geneQuery.trim().split(/\s+/);
+        return this.alterationEnrichmentRowData.filter(gene => {
+            return newGenes.includes(gene.hugoGeneSymbol);
+        });
+    }
+
+    @computed get genesWithMaxFrequency(): AlterationEnrichmentRow[] {
+        if (
+            this.alterationsEnrichmentData.isComplete &&
+            this.alterationsEnrichmentAnalysisGroups.isComplete
+        ) {
+            let genesWithMaxFrequency: AlterationEnrichmentRow[] = [];
+            let usedGenes: number[] = [];
+            let alterationRowData: AlterationEnrichmentRow[] = getAlterationRowData(
```

Comment:
These all should be `const`
```suggestion
            const genesWithMaxFrequency: AlterationEnrichmentRow[] = [];
            const usedGenes: number[] = [];
            const alterationRowData: AlterationEnrichmentRow[] = getAlterationRowData(
```